### PR TITLE
php version.

### PR DESCRIPTION
**Pull request by digi3colin**

### CODE REVIEW

**Comment by gaplo917** [comment]
@digi3colin can you rename your files and split into two folder ?

Suggestion

```
php-9-for-loop/abcedfghppp.php
php-permutation/abcedfghppp.php
```

you can also think of a better name to describe instead of my suggestion :+1:

File: php/index.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            }
+          }
+        }
+      }
+    }
+    echo 'search '.$count.' times';
+}
+
+pure_brute_force();
+?>
```
Comment (cherrry) [inline php/index.php]: http://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag

Comment (digi3colin) [inline php/index.php]: thanks cherrry's advice. The close tag omitted.

**Comment by gaplo917** [comment]
Please rebase & squash your commit into single commit  ( PHP related )

Similar case
#24

**Comment by digi3colin** [comment]
is it helped?
The commits has been rebased.
Thanks for the advice!

**Pull request merged**